index.js

```diff
@@ -272,6 +272,12 @@ alexa.session = function(session) {
 alexa.apps = {};
 
 alexa.app = function(name, endpoint) {
+  "use strict"
+
+  if (!(this instanceof alexa.app)) {
+      throw new Error("Function needs to be called with the new keyword");
```

Comment:
Maybe needs should be must. So just "Function must be called with the new keyword".